src/components/quickStarts.stories.mdx

```diff
@@ -2,83 +2,19 @@
 
 # Getting Started
 
-## Setup the Environment
+This section introduces the advanced features of the Amazon Chime SDK React Components Library.
```

Comment:
Suggest to re-phrase a bit:

"This section explains how to use Amazon Chime SDK React Components Library for few advanced use-cases."

Reply:
a bit confuse. Would not associate advanced use case with quick start :)

Reply:
I won't mention "advanced", so how about:
"This section explains how to use the Amazon Chime SDK React Components Library for some common use-cases."

Reply:
Since we are adding more and more content into it. I think "Common Use Cases" could be better. "QuickStarts" is Vidya's suggestion.

It's only a doc, so I think we can update. I just need to update all the old links.

Reply:
I will submit a separate PR to update the "QuickStarts" to "Use cases".